The index.d.ts does not work

Please:

- [X] Check for duplicate issues. Please file separate requests as separate issues on GitHub.
- [x] Describe how to reproduce the bug.
run the backend/tests/imageRepTest.ts for example
the import is still not recognized properly
- [X] Use the latest version, if possible. Always indicate which version you are using. 
Current main
- [X] Include error messages or gifs/screenshots of problematic behavior, if applicable.
`TSError: ⨯ Unable to compile TypeScript:
src/representative.ts:5:24 - error TS7016: Could not find a declaration file for module 'colorthief'. 'Y:/Repos/HIE - dev/Hierarchical-Image-Explorer/node_modules/colorthief/dist/color-thief.js' implicitly has an 'any' type.`
- [ ] Provide an example.

--- a/backend/tsconfig.json
+++ b/backend/tsconfig.json
@@ -15,5 +15,11 @@
   "exclude": [
     "node_modules",
     "<node_internals>/**",
-  ]
+  ],
+  "files":[
+    "src/index.d.ts"
+  ],
+  "ts-node": {
+    "files":true
+  }
 }

